RZE/Engine/Src/EngineCore/Resources/ResourceHandler.h

```diff
@@ -87,10 +89,13 @@ class ResourceHandler
 	// @TODO This should be removed when resource instance layer goes in.
 	// Do not rely on this function
 	ResourceHandle Make(const std::string& resourceID, IResource* resource);
+    const std::unordered_map<std::string, ResourceSource>& GetResourceTableInternal() const;
```

Comment:
I had to make this public, IDK why adding a friend class to the editor panel wasn't working, but I'm too tired 😭

Reply:
also fuck me this isn't tabs